components/AuthenticatedPage.js

```diff
@@ -73,7 +73,12 @@ class AuthenticatedPage extends React.Component {
 
     return (
       <Page noRobots {...pageProps}>
-        {this.renderContent(loadingLoggedInUser, LoggedInUser)}
```

Comment:
It looks like the first thing we do in `renderContent` is `if (!LoggedInUser)`. The function seems designed to handle this behavior. Why was it not working with the previous version?

Reply:
@Betree : Ops, you are right. This condition is not necessary; I didn't see we already check it in the function. The only thing that needed to be removed was the graphql skip condition. 😄